Source/DiabloUI/selyesno.cpp

```diff
@@ -20,15 +20,13 @@ void selyesno_Free()
 {
 	ArtBackground.Unload();
 
-	for (std::size_t i = 0; i < vecSelYesNoDialogItems.size(); i++) {
-		UiListItem *pUIListItem = vecSelYesNoDialogItems[i];
+	for (auto pUIListItem : vecSelYesNoDialogItems) {
 		if (pUIListItem)

```

Comment:
If by "they" you mean me then sure since I originally wrote our implementation of DiabloUI as a clean room implementation :P
Only the exe code matches the original implementation. DiabloUI was probably the biggest issue with the game, pretty simple to document, and so not worth figuring out the exact implementation of.